internal/storage/storage_step2_timeout_test.go

```
@@ -0,0 +1,95 @@
+		t.Skipf("mkfifo unsupported: %v", err)
+	}
+
+	releaseWriter := make(chan struct{})
+	writerDone := make(chan struct{})
+	go func() {
+		defer close(writerDone)
+		w, err := os.OpenFile(fifo, os.O_WRONLY, 0)
+		if err != nil {
+			return
+		}
+		defer w.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `w.Close` is not checked

(errcheck)